includes/admin/assets.php

```diff
@@ -129,5 +130,7 @@ public function load()
 			wp_enqueue_style('sc-welcome-style', $css_path . 'sc-welcome-pg-style.min.css', [], SIMPLE_CALENDAR_VERSION);
 			wp_enqueue_style('sc-tail-style', $css_path . 'tailwind.min.css', [], SIMPLE_CALENDAR_VERSION);
 		}
+		wp_enqueue_style('sc-tail-style', $css_path . 'tailwind-output.css', [], SIMPLE_CALENDAR_VERSION);
```

Comment:
We don't need to enqueue this style as we alredy included tailwind.min file.

Reply:
i had removed this enqueue.